components/NavBar.tsx

```diff
@@ -1,28 +1,45 @@
+"use client";
 import Image from "next/image";
 import Link from "next/link";
-import { Lobster } from "next/font/google";
+import { Lobster, Inter } from "next/font/google";
+import { useSession } from "next-auth/react";
 
 const lobster = Lobster({ weight: "400", subsets: ["latin"] });
```

Comment:
looks like this was added a while ago, but you don't want to use fonts like this. You should put this in the root layout like the inter font is and configure tailwind to support this font with a class.

all your styling should be done through tailwind classes

[the next font docs have a section on using their font API with tailwind](https://nextjs.org/docs/app/building-your-application/optimizing/fonts#with-tailwind-css)